composer.json

```diff
@@ -96,7 +96,8 @@
         "egulias/email-validator": "~1.2,>=1.2.8|~2.0",
         "symfony/phpunit-bridge": "~3.4|~4.0",
         "symfony/security-acl": "~2.8|~3.0",
-        "phpdocumentor/reflection-docblock": "^3.0|^4.0"
+        "phpdocumentor/reflection-docblock": "^3.0|^4.0",
+        "swiftmailer/swiftmailer": "^6.0@dev"
```

Comment:
why adding this one? it's not used in the code base so I wouldn't add it to please phpstan :)

Reply:
https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Monolog/Handler/SwiftMailerHandler.php is using it.

Reply:
it's not: there is no test case for it, so the code just doens't run - so there is no need to add this in require-dev (it's not required)
unless you want to add a test case :)